frontend-react/src/components/AuthElement.tsx

```diff
@@ -33,11 +34,12 @@ export const AuthElement = ({
     );
     // Dynamically authorize user from single or multiple allowed user types
     const authorizeMemberType = useMemo(() => {
```

Comment:
out of scope of this change, so ignore if you want, but since this is just a memoized value, could it be `memberTypeAuthorized`?